### ISSUE TITLE
reactive-cli 1.3.2 rp fails with "symbol not found" on Mac

### ISSUE THREAD

**Issue by ecausarano** [lightbend/reactive-cli#186]
On fully up to date brew, installed cli and found this failure

```
⇒  rp
dyld: lazy symbol binding failed: Symbol not found: __ZNK3re23RE223NumberOfCapturingGroupsEv
  Referenced from: /usr/local/bin/rp
  Expected in: /usr/local/opt/re2/lib/libre2.0.dylib

dyld: Symbol not found: __ZNK3re23RE223NumberOfCapturingGroupsEv
  Referenced from: /usr/local/bin/rp
  Expected in: /usr/local/opt/re2/lib/libre2.0.dylib
```

seems that https://homebrew.bintray.com/bottles/re2-20190101.mojave.bottle.tar.gz doesn't work

**Comment by eed3si9n** [comment]
Thanks for the report @ecausarano.

**Comment by eed3si9n** [comment]
Looks like https://github.com/google/re2/commit/2cf86e5ab6dcfe045a1f510c2b9a8b012a4158cd#diff-a60a8d25ed15adf68b94c85775fd3cf7 removed a method in use.

A workaround is to list `brew info re2` and switch to a previous one one your machine, if you have one e.g. `brew switch re2 20181001`.

**Comment by eed3si9n** [comment]
I just released reactive-cli 1.6.0.
`brew update && brew upgrade reactive-cli` should fix this issue.

**Comment by ecausarano** [comment]
👍 thanks
